src/main/java/com/example/cargive/domain/favorite/controller/FavoritePkInfoController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.cargive.domain.favorite.controller;
+
+import com.example.cargive.domain.favorite.infra.query.dto.FavoriteQueryResponse;
+import com.example.cargive.domain.favorite.service.FavoritePkInfoService;
+import com.example.cargive.global.base.BaseResponse;
+import com.example.cargive.global.base.BaseResponseStatus;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/favorite/info")
+public class FavoritePkInfoController {
+    private final FavoritePkInfoService favoritePkInfoService;
+
+    @GetMapping("/{favoriteGroupId}") // 즐겨찾기된 주차장 조회
+    public BaseResponse<FavoriteQueryResponse> getFavoritePkInfos(@PathVariable Long favoriteGroupId,
+                                                                  @RequestParam String sortBy,
+                                                                  @RequestParam Long memberId,
+                                                                  @RequestParam int page) {
+        return new BaseResponse<>(BaseResponseStatus.SUCCESS,
```

Comment:
제가 BaseResponse.java line 23 보시면, 대부분의 DTO를 반환하는 API가 http 상태코드 200을 반환할 것이므로 해당 상태코드를 굳이 작성하지 않도록 생성자를 만들어두었습니다. 따라서 다음과 같이 수정해도 같은 의미로 정상적으로 동작할 것으로 보여요!
```suggestion
        return new BaseResponse<>(
```

Reply:
감사합니다! 해당 사항 수정하여 새로 커밋하겠습니다!